Activities/Database/UiPath.Database.Activities/ConnectionHelper.cs

```diff
@@ -30,7 +30,7 @@ public static void ConnectionValidation(DatabaseConnection existingConnection, S
                 {
                     throw new ArgumentException(Resources.ValidationError_ConnectionStringMustBeSet);
                 }
-                if (connString == null && existingConnection == null)
+                if (connString == null && connSecureString == null)
```

Comment:
What happens if existing connection is not null? It will not work with existing connection

Reply:
That if is inside another if: if (existingConnection == null), so the existingConnection can't be not null there.